python/cmd/data.py

```diff
@@ -42,6 +42,23 @@
 from argparse import ArgumentParser
 import importlib
 
+# Cleaner way to handle module import paths irrespective of current working
+# dir in the housing-insights project folder
+import sys, os
+path = ""
+
+# get absolute path for housing-insights or housing-insights/python
+# which ever is closest
+while os.path.basename(os.path.abspath(path)) not in ["housing-insights",
+                                                      "python"]:
+    path += "../"
+
+# if at housing-insights folder, move down to housing-insights/python
+if os.path.basename(os.path.abspath(path)) == "housing-insights":
+    path += "python/"
+
+sys.path.append(os.path.abspath(path))
+
```

Comment:
So here's some code I pulled from another project - a one-liner that you can use to always get the directory path of the repository, which you can then join to whatever relative path you need. 

`repo_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))`

You'll need to edit the number of os.pardir as necessary. But, using __file__ instead of os.path.basename means it uses this file's path, not the path it was run from. 

I'll probably try to make a universal solution for this and go in and edit it in all the script files so it's done the same everywhere - if this is working, no need to edit for now.